.ticket/scripts/ticket-report.sh

```
@@ -0,0 +1,124 @@
+TOTAL_TESTING=$(echo "$ALL_ACTIVE_TICKETS" | xargs -I {} grep -l "^status: testing" {} 2>/dev/null | wc -l)
+TOTAL_COMPLETED_IN_TICKETS=$(echo "$ALL_ACTIVE_TICKETS" | xargs -I {} grep -l "^status: completed" {} 2>/dev/null | wc -l)
+TOTAL_COMPLETED_ARCHIVED=$(echo "$ALL_ARCHIVED_TICKETS" | wc -l)
+TOTAL_COMPLETED=$((TOTAL_COMPLETED_IN_TICKETS + TOTAL_COMPLETED_ARCHIVED))
+TOTAL_BLOCKED=$(echo "$ALL_ACTIVE_TICKETS" | xargs -I {} grep -l "^status: blocked" {} 2>/dev/null | wc -l)
+
+TOTAL_ACTIVE=$((TOTAL_TODO + TOTAL_IN_PROGRESS + TOTAL_REVIEW + TOTAL_TESTING + TOTAL_BLOCKED))
+TOTAL_ALL=$((TOTAL_ACTIVE + TOTAL_COMPLETED))
+
+echo "| Status | Count | Percentage |"
+echo "|--------|-------|------------|"
+printf "| Todo | %d | %.1f%% |\n" $TOTAL_TODO $(echo "scale=1; $TOTAL_TODO * 100 / $TOTAL_ALL" | bc)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)